lib/rules/function-name-case/README.md

```diff
@@ -124,36 +124,29 @@ For example, with `"lower"`.
 Given:
 
 ```json
-["some-function", "/^get.*$/"]
+["--some-function", "/^--get.*$/"]
 ```
 
 The following patterns are considered problems:
```

Comment:
[ask] Do we need to give examples of considered problems in `ignoreFunctions`?

Reply:
Narh, we don't need to. We can just show the rule being made more permissive. For example, by showing a pattern that was a problem previously without the option being used:

e.g. before the secondary options header we can show `--FOO()` being a problem, and then:

Given:

```
["--FOO"]
```

It no longer being an problem. 

I think we can just take the problem section out in this pull request, and save the suggestions above (showing the permissiveness and using `foo`) for another issue/pr.

Reply:
Thanks for the explanation. I understand and agree. 👍🏼
Then, I'll keep the PR change as-is.